### ISSUE TITLE
Regression of comment collapse performance improvements

### ISSUE THREAD

**Issue by SrivatsanSenthilkumar** [aeharding/voyager#895]
**Describe the bug**
Through #848, you had improved the comment collapse performance as a part of the 1.18.0 version update but it seems that has regressed with succeeding updates. The animation for all the comments collapsing is occurring instead of only for the collapsed comment like intended by #848

**To Reproduce**

1. Go to a thread with a lot of comments
2. Collapse the top comment and see the old stuttery animation
3. Comment thread - https://lemmy.world/comment/5055451

**Expected behavior**
Collapse smoothly like it did in version 1.18

**Screenshots**
https://github.com/aeharding/voyager/assets/141351084/f0a3f339-8b01-4a06-b519-613c2151c1c0

**Smartphone (please complete the following information):**

- Device: Moto G60
- OS: Android 12
- Native app 
- 1.19.3 

**Additional context**

**Comment by SrivatsanSenthilkumar** [comment]
It is weird. Some(probably most) comments collapse properly respecting the changes from #848 but a very few dont

**Comment by aeharding** [comment]
Hi @SrivatsanSenthilkumar since you're on Android, can you go through the Github Releases, download the APK and test them out starting on version 1.18 and tell me when the performance regressed? Then it will be a lot easier for me to narrow down. :) If you can, whenever you have time.

Thank you!

**Comment by SrivatsanSenthilkumar** [comment]
Sure I will do this. Just give me 2 days

**Comment by SrivatsanSenthilkumar** [comment]
@aeharding 

Looks like I won't need the 2 days. I just installed 1.18.0 to try it out and I found the comment collapses with stutters in the same way as in the  video above. I tried collapsing other threads and observed that this problem is shared by comments with a lot of child comments where at least a few of the child comments are long. Seems like these long comments also render the collapse animation even though they were not the comment tapped on to collapse. If I understand correctly (and I may not be understanding correctly as I am not a developer) isn't this supposed to be prevented by #848
